src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -643,31 +644,6 @@ public void testClientMixedMemoryJvmParameteres() throws Exception {
     }
   }
 
-  /**
```

Comment:
These tests are just getting moved to the "Latest" folder. I added the logic to check that the heartbeatFrequency variable in SFBaseSession is reflecting the changed value. This failed the old driver tests because the heartbeatFrequency variable didn't used to get properly set in old driver versions.